holoocean:
  holoocean_node:
    ros__parameters:
      draw_arrow: true
      show_viewport: true
      relative_path: true
      scenario_path: 'config/torpedo_3fin.json'
      publish_commands: true
      
